applications/registration-api/src/main/resources/application.yml

```diff
@@ -39,38 +39,15 @@ spring:
       clusterNodes: ${FDK_ES_CLUSTERNODES:elasticsearch5:9300}
       clusterName: ${FDK_ES_CLUSTERNAME:elasticsearch}
 
-saml:
-  sso:
-    metadata-generator:
-      entity-id: difi.brreg.datakatalog.registrering.${registrationApi_OpenshiftEnvironment:dev}
-    idp:
-      metadata-location: file:${registrationApi_idportenMetadataFile:conf/idporten/idporten-ver2.difi.no-v3_signed_meta.xml}
-      localMetadataLocation: file:conf/idporten/difi.brreg.datakatalog.registrering.${registrationApi_OpenshiftEnvironment:dev}.xml
-    metadata-manager:
-      refresh-check-interval: -1
-    key-manager:
-      public-key-pem-location: file:conf/idporten/difi.brreg.datakatalog.registrering.${registrationApi_OpenshiftEnvironment:dev}.xml
-      store-location: file:conf/idporten/difi.brreg.datakatalog.registrering.${registrationApi_OpenshiftEnvironment:dev}.jks
-      store-pass: ${registrationApi_ipStorePassword}
-      default-key: difi.brreg.datakatalog.registrering.${registrationApi_OpenshiftEnvironment:dev}
-      key-passwords:
-        difi.brreg.datakatalog.registrering.it1: ${registrationApi_ipKeyPassword}
-        difi.brreg.datakatalog.registrering.st1: ${registrationApi_ipKeyPassword}
-        difi.brreg.datakatalog.registrering.ppe: ${registrationApi_ipKeyPassword}
-        difi.brreg.datakatalog.registrering.prd: ${registrationApi_ipKeyPassword}
-        difi.brreg.datakatalog.registrering.tt1: ${registrationApi_ipKeyPassword}
-        difi.brreg.datakatalog.registrering.ut1: ${registrationApi_ipKeyPassword}
-        difi.brreg.datakatalog.registrering.st2: ${registrationApi_ipKeyPassword}
-        difi.brreg.datakatalog.registrering.dev: ${registrationApi_ipKeyPassword}
-    context-provider:
-      lb:
-        context-path: /
-        include-server-port-in-request-url: ${registrationApi_IncludeServerPortInRequestUrl:true}
-        scheme: https
-        server-name: ${registrationApi_ServerName:localhost}
-        server-port: ${registrationApi_ServerPort:8099}
-    logout:
-      default-target-url: /
+rest.security.issuer-uri: http://sso:8084/auth/realms/fdk
+
+security:
+  oauth2:
+    resource:
+      #TODO FIX NOW!
+      id: fdk-registration-api
+      #curl http://sso:8084/auth/realms/fdk | |jq .public_key -r
+      jwt.key-value: "-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEArC2L2Ti8zj6nhMaUDVHTOYpIe0w8Nwn02qQqoHtT5vjT3lA+qLk8iV4NXTEOYV8Q2j2PFsQCaSvL/Phw6CJa08tY/AUxzcmFVYw9iJJVedrcCypsQGhuNGG39MU372Gp+p8oyLg4UxJVRShPOZIskGCiplZNocFibYhDK8WtsgvH7jbvW0K1HtsGk3R3IajmAO53a2LktjB0BxNi9sZ9x9lCMhOU64eotaWMnsQE7Tk7ETJIhm1PMbwtStkFP/U05dhxJA5J6rw+q+srigKSu49DPUvbZtJqahnDzS3RQE5JewmqxnwT18R6QPFiURnZkYbxlJ1GhKrt5lRwEqAEhQIDAQAB\n-----END PUBLIC KEY-----"
```

Comment:
As we discussed, fetching public key before application start is better than having a hard-coded value here. Note that developers should also be able to run this application locally (i.e. not in container). Therefore, it may be a good idea to fetch public key in Java rather than in Docker container.